src/NUnitFramework/framework/Constraints/ThrowsExceptionConstraint.cs

```diff
@@ -22,7 +22,11 @@
 // ***********************************************************************
 
 using System;
+
+#if ASYNC
+using System.Threading.Tasks;
 using NUnit.Framework.Internal;
```

Comment:
Should this be inside the #if/#endif?  It was not previously.

Reply:
Yes. It was only needed by the original conditional ASYNC section.